prusti-viper/src/encoder/errors/error_manager.rs

```diff
@@ -696,6 +698,20 @@ impl<'tcx> ErrorManager<'tcx> {
                 )
             }
 
+            ("refute.failed:refutation.true", ErrorCtxt::Panic(PanicCause::Refute)) => {
+                PrustiError::verification(
+                    "the refuted expression holds in all cases or could not be reached",
+                    error_span,
+                )
+            }
+
+            ("assert.failed:assertion.false", ErrorCtxt::Panic(PanicCause::Refute)) => {
+                PrustiError::verification(
+                    "the refuted expression holds in all cases or could not be reached",
+                    error_span,
+                )
+            }
```

Comment:
@simon-hrabec are both of these needed? Won't Viper always return either `"assert.failed:assertion.false"` or `"refute.failed:refutation.true"` (I suspect the latter)?